scripts/release-tools/jira.sh

```diff
@@ -163,6 +168,8 @@ print_ticket_verification_summary() {
 }
 
 name2slack() {
+  echo "${1%%@*}"
+  return
```

Comment:
This is okay to use the default Slack handle, however some colleagues choose to change the handles to something more unique and that is where the manual mapping would help (after fixing it).

A dry-run has recognized only some colleagues:
<img width="568" alt="Screenshot 2022-09-05 at 16 26 33" src="https://user-images.githubusercontent.com/114479/188471443-df7ca06a-b7ac-4ed9-9159-7d827a364193.png">

Reply:
When I click on the question mark (or the name of the user), I see various numbers of suggestions - e.g., 1 for Allan, but 4 for Boaz.
If I simply hit enter without checking the question marks, then the users who have question marks are not mentioned.